src/fileoverview_comment_transformer.ts

```diff
@@ -22,13 +23,20 @@ const FILEOVERVIEW_COMMENT_MARKERS: ReadonlySet<string> =
  * Given a parsed \@fileoverview comment, ensures it has all the attributes we need.
  * This function can be called to modify an existing comment or to make a new one.
  *
+ * @param options Compiler options.
```

Comment:
Leave out this line, because it just restates the type of the param

Reply:
Done.